configure.in

```
@@ -678,6 +678,7 @@ WX_ARG_ENABLE(extended_rtti, [  --enable-extended_rtti  use extended RTTI (XTI)]
 WX_ARG_DISABLE(optimise,   [  --disable-optimise      compile without optimisations], wxUSE_OPTIMISE)
 
 WX_ARG_ENABLE(profile,       [  --enable-profile        create code with profiling information], wxUSE_PROFILE)
+WX_ARG_DISABLE(pic,          [  --disable-pic           disable position independent code], wxUSE_PIC)
```

```suggestion
WX_ARG_DISABLE(pic,          [  --disable-pic           don't use position independent code when building static libraries (shared libraries always use PIC)], wxUSE_PIC)
```
This might be too long, but I think the current message is a bit misleading...

Alternatively, we could give an error if `--disable-pic` is specified, but `--disable-shared` is not, what do you think?

---

I'll update the description.
I can add a warning, like is done for `wxUSE_PLUGINS`.